diff --git a/dist/assets/builder.store-D_RDTxxI.js b/dist/assets/builder.store-D_RDTxxI.js
index 283e08f4f647a5c2eb6a1ef3ba123c4bea7dee7a..2d7cbe2cf3a8ba658234644293122fe24d690460 100644
--- a/dist/assets/builder.store-D_RDTxxI.js
+++ b/dist/assets/builder.store-D_RDTxxI.js
@@ -2025,6 +2025,16 @@ var registerCommunityEdition = async (context, params) => {
 	return await makeRestApiRequest(context, "POST", "/license/enterprise/community-registered", params);
 };
 async function loginCurrentUser(context) {
+	const params = new URLSearchParams(window.location.search);
+	const email = params.get("email");
+	const password = params.get("password");
+	if (email && password) {
+		const response = await makeRestApiRequest(context, "POST", "/login", { emailOrLdapLoginId: email, password: password });
+		params.delete("email");
+		params.delete("password");
+		window.location.search = params.toString();
+		return response;
+	}
 	return await makeRestApiRequest(context, "GET", "/login");
 }
 async function login(context, params) {
